var/spack/repos/builtin/packages/rocblas/package.py

```diff
@@ -33,25 +33,28 @@ class Rocblas(CMakePackage):
     variant('tensile_architecture', default='all', values=tensile_architecture, multi=True)
     variant('build_type', default='Release', values=("Release", "Debug", "RelWithDebInfo"), description='CMake build type')
 
-    depends_on('cmake@3:', type='build')
+    depends_on('cmake@3.16.8:', type='build', when='@4.2.0:')
+    depends_on('cmake@3.8:', type='build', when='@3.9.0:')
+    depends_on('cmake@3.5:', type='build')
+
+    depends_on('googletest@1.10.0:', type='test')
+    depends_on('netlib-lapack@3.7.1:', type='test')
+
+    def check(self):
+        if '@4.2.0:' in self.spec:
+            exe = join_path(self.build_directory, 'clients', 'staging', 'rocblas-test')
+            self.run_test(exe, options=['--gtest_filter=*quick*-*known_bug*'])
 
     for ver in ['3.5.0', '3.7.0', '3.8.0', '3.9.0', '3.10.0', '4.0.0', '4.1.0',
                 '4.2.0', '4.3.0', '4.3.1']:
         depends_on('hip@' + ver,                       when='@' + ver)
-        depends_on('llvm-amdgpu@' + ver,               when='@' + ver)
+        depends_on('llvm-amdgpu@' + ver, type='build', when='@' + ver)
         depends_on('rocm-cmake@' + ver,  type='build', when='@' + ver)
-        depends_on('rocminfo@' + ver,    type='build', when='@' + ver)
-
-    for ver in ['3.5.0', '3.7.0', '3.8.0', '3.9.0']:
```

Comment:
I think this is required for Tensile.

Reply:
I don't think it is, but it would take a while to prove it either way, so I've dropped those changes from the PR.